src/app/containers/CpsRecommendations/RecommendationsPromo/index.jsx

```diff
@@ -32,21 +107,26 @@ const RecommendationsPromo = ({ promo, dir, eventTrackingData }) => {
       }}
       enableGelGutters
     >
-      <StyledStoryPromoWrapper data-e2e="story-promo-wrapper">
-        <StoryPromo
-          item={promo}
-          dir={dir}
-          isRecommendation
-          displaySummary={false}
-          eventTrackingData={eventTrackingData}
-        />
-      </StyledStoryPromoWrapper>
+      <StyledPromoWrapper data-e2e="story-promo-wrapper">
+        <ImageWrapper>
+          <RecommendationsImage indexImage={indexImage} lazyLoad />
+        </ImageWrapper>
+        <TextWrapper>
+          <StyledHeadline script={script} service={service}>
+            <Link
+              href={url}
+              onClick={eventTrackingData ? handleClickTracking : null}
```

Comment:
This is me being pedantic but would it better (readability wise) to create another variable above?

Reply:
```suggestion
              onClick={eventTrackingData && handleClickTracking}
```